app/src/main/java/com/paris_2/san3a/data/repository/LocationRepositoryImp.kt

```diff
@@ -0,0 +1,34 @@
+package com.paris_2.san3a.data.repository
+
+import com.paris_2.san3a.data.mapper.toCities
+import com.paris_2.san3a.data.mapper.toStates
+import com.paris_2.san3a.data.source.remote.location.LocationRemoteDataSource
+import com.paris_2.san3a.data.source.remote.location.request.CitiesRequest
+import com.paris_2.san3a.domain.NoCitiesFoundException
+import com.paris_2.san3a.domain.NoGovernmentsFoundException
+import com.paris_2.san3a.domain.entity.Cities
+import com.paris_2.san3a.domain.entity.States
+import com.paris_2.san3a.domain.repository.LocationRepository
+
+class LocationRepositoryImp(
+    private val locationRemoteDataSource: LocationRemoteDataSource,
```

Comment:
Do you think that we need to separate the location to be in a separate repository while the location is part of the flow of registration?